### PR TITLE
Add tabletSupport flag, provision wording

### PR DESCRIPTION
**Pull request by mlsteele**

### CODE REVIEW

File: shared/devices/add-device/index.tsx [hunk 1]
```diff
@@ -119,7 +121,7 @@ const styles = Styles.styleSheetCreate(() => ({
     borderStyle: 'solid',
     borderWidth: 1,
     padding: Styles.globalMargins.tiny,
-    width: Styles.isMobile ? 160 : 140,
+    width: Styles.isMobile ? 192 : 168,
```
Comment (chrisnojima) [inline shared/devices/add-device/index.tsx]: why this change

File: shared/util/feature-flags.native.tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ const ff: FeatureFlags = {
   dbCleanEnabled: false,
   fastAccountSwitch: true,
   foldersInProfileTab: false,
+  tabletSupport: __DEV__,
```
Comment (chrisnojima) [inline shared/util/feature-flags.native.tsx]: we never do this as you'll never see it in testflight

File: shared/util/feature-flags.desktop.tsx [hunk 3]
```diff
@@ -33,6 +34,7 @@ const inAdmin: {[K in keyof FeatureFlags]?: boolean} = {
   chatIndexProfilingEnabled: true,
   connectThrashCheck: true,
   dbCleanEnabled: true,
+  tabletSupport: true,
```
Comment (chrisnojima) [inline shared/util/feature-flags.desktop.tsx]: this doesn't make sense, this is for desktop

Comment (mlsteele) [reply shared/util/feature-flags.desktop.tsx]: this is meant to mean whether tablet support is public, not whether the current device is a tablet

File: shared/util/feature-flags.native.tsx (hunk 2 above)
Comment (mlsteele) [reply shared/util/feature-flags.native.tsx]: that's enough to get started for now. What should be done? I saw `inAdmin` in feature-flags.desktop

File: shared/devices/add-device/index.tsx (hunk 1 above)
Comment (mlsteele) [reply shared/devices/add-device/index.tsx]: it wrapped "or tablet"

File: shared/util/feature-flags.native.tsx (hunk 2 above)
Comment (chrisnojima) [reply shared/util/feature-flags.native.tsx]: just use true

File: shared/__mocks__/feature-flags.tsx [hunk 4]
```diff
@@ -14,13 +14,14 @@ const ff: FeatureFlags = {
   dbCleanEnabled: false,
   fastAccountSwitch: true,
   foldersInProfileTab: true,
-  lagRadar: false,
+  lagradar: false,
```
Comment (chrisnojima) [inline shared/__mocks__/feature-flags.tsx]: ?????

File: shared/__mocks__/feature-flags.tsx [hunk 5]
```diff
@@ -14,13 +14,14 @@ const ff: FeatureFlags = {
...
   moveOrCopy: true,
   newTeamBuildingForChatAllowMakeTeam: false,
   openTeamSearch: true,
   outOfDateBanner: true,
   proofProviders: true,
   stellarExternalPartners: false,
+  tabletsupport: false,
```
Comment (chrisnojima) [inline shared/__mocks__/feature-flags.tsx]: `tabletSupport`

**Comment by mlsteele** [comment]
ready and typo free
